data/test-cases/plugins/github/subscriptions/Webhook_check.md

```diff
@@ -0,0 +1,42 @@
+---
+# (Required) Ensure all values are filled up
+name: "An ephemeral post is generated in the channel or DM/GM if the webhook is not found on the github side."
```

Comment:
Please try to cut short these names as we might face some issues in executing these from Zephyr.